### PR TITLE
Remove fieds migrated to the core

### PR DESCRIPTION
**Pull request by maximebiloe**
Related to http://forge.prestashop.com/browse/FF-95

**Don't merge before 1.7.3.0**

### CODE REVIEW

File: ps_legalcompliance.php [hunk 1]
```diff
@@ -224,7 +224,7 @@ public function createConfig()
         $langs = $langs_repository->findAll();
 
         foreach ($langs as $lang) {
-            $custom_cart_text_values[(int) $lang->id] = $this->trans('The order will only be confirmed when you click on the button \'Order with an obligation to pay\' at the end of the checkout!', array(), 'Modules.Legalcompliance.Shop');
+            $custom_cart_text_values[(int) $lang->id] = $this->trans('The order will only be confirmed when you click on the button \'Order with an obligation to pay\' at the end of the checkout!', array(), 'Modules.Legalcompliance.Shop', $lang->locale);
```
Comment (aleeks) [inline ps_legalcompliance.php]: :+1:

File: upgrade/upgrade-3.0.0.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *  International Registered Trademark & Property of PrestaShop SA
+ */
+
+if (!defined('_PS_VERSION_'))
+    exit;
+
+function upgrade_module_3_0_0($module)
+{
+    Configuration::deleteByName('AEUC_LABEL_DELIVERY_TIME_AVAILABLE');
+    Configuration::deleteByName('AEUC_LABEL_DELIVERY_TIME_OOS');
```
Comment (Quetzacoalt91) [inline upgrade/upgrade-3.0.0.php]: We could use this upgrade file to replace the core default values by potential customized string from the merchant, don't we ?

Comment (Quetzacoalt91) [reply upgrade/upgrade-3.0.0.php]: Just found my answer. This is already done in https://github.com/PrestaShop/PrestaShop/pull/8220/files#diff-62345160fc5872e6e5fa50f38f8044ffR42

**Comment by eternoendless** [comment]
Thank you @maximebiloe

**Pull request merged**